### PR TITLE
feat: ability to create a tx and submit it to multiple chains

### PR DESCRIPTION
**Pull request by 0xtekgrinder**

### CODE REVIEW

File: helpers/createTx.sh [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+        echo "No chain provided"
+        exit 1
+    fi
+
+    for chain in $(echo $chains | sed "s/,/ /g")
+    do
+        echo ""
+        echo "Running on chain $chain"
+        uri=$(chain_to_uri $chain)
+        export CHAIN_ID=$(chain_to_chainId $chain)
```
Comment (GuillaumeNervoXS) [inline helpers/createTx.sh:124]: It will be written on the .env?

**Comment by GuillaumeNervoXS** [comment]
Really great!!

On the next steps is maybe to provide arguments, but we will try to it for a particular use case. I'll see with the governanc escript is it can be useful.

**Comment by 0xtekgrinder** [comment]
It should be pretty easy to add arguments so tell me and I can implement it

File: helpers/createTx.sh [hunk 2]
```diff
@@ -0,0 +1,188 @@
...
+  echo -e "chain: chain(s) to run the script on (separate with commas)"
+  echo -e "\t1: Ethereum Mainnet"
+  echo -e "\t2: Arbitrum"
+  echo -e "\t3: Polygon"
+  echo -e "\t4: Gnosis"
+  echo -e "\t5: Avalanche"
+  echo -e "\t6: Base"
+  echo -e "\t7: Binance Smart Chain"
+  echo -e "\t8: Celo"
+  echo -e "\t9: Polygon ZkEvm"
```
Comment (sogipec) [inline helpers/createTx.sh:94]: Can you plz add Linea as well

**Review by GuillaumeNervoXS — APPROVED** [review]

**Pull request merged**